keyboards/kinesis/kintwin/info.json

```diff
@@ -0,0 +1,250 @@
+{
+    "development_board": "blackpill_f401",
+    "keyboard_name": "Kinesis",
+    "url": "https://github.com/alvicstep/kintwin",
+    "manufacturer": "https://github.com/alvicstep/kintwin",
+    "maintainer": "Alex Stepanov",
+    "usb": {
+        "device_version": "0.0.3",
+        "vid": "0xFEED",
+        "pid": "0x4B4E",
+        "shared_endpoint": {
+            "keyboard": true   
+        } 
+    },
+    "features": {
+        "bootmagic": true,
+        "console": false,
+        "command": false,
+        "mousekey": false,
+        "extrakey": false,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "rows": ["A0", "A1", "A2", "A3", "A4", "A5", "A6"],
+        "cols": ["A7", "B0", "B1", "B10", "B9", "B8", "B7", "B14", "B13", "B12", "C14", "C15", "B15", "B6", "B5"]
+    },
+    "diode_direction": "ROW2COL",
+    "indicators": {
+        "caps_lock": "A15",
+        "scroll_lock": "B3",
+        "num_lock": "B4",
+        "compose": "C13",
+        "on_state": 0
+    },    
+    "layouts": {
+        "LAYOUT": {
```

Comment:
Please remove this layout and replace it with `LAYOUT_pretty`.

Reply:
"Layout" matrix was added to have the widest compatibility with the existing keymaps. For example, if I remove "layout" matrix then the controller will stop working with "default" keymap. 

Also, other controllers like kint41 or stapelberg and others implement both matrices.

Can you please explain your reasoning for removing "layout" matrix or did I misunderstand your intention?

Reply:
See this PR: https://github.com/qmk/qmk_firmware/pull/21384

Reply:
I would like to understand the request better. 

You would like to:
1. remove LAYOUT
2. create LAYOUT alias to point to LAYOUT_pretty

       Something like that:
          "layout_aliases": {
              "LAYOUT": "LAYOUT_pretty"
          },

Is it correct?

I looked at the PR you referenced. "Split" layout (or layout_pretty in our case) is easier to read and as you put it, it has "more conventionally correct ordering". And I think it is very reasonable rule to follow for new keyboards which have no existing keymaps from other contributors. And if you insist I will definitely remove "stack" layout from kintwin controller configuration. However, as I already mentioned, this will make this controller incompatible with a lot of keymaps. 

In fact, I went through every keymap under "keymaps" directory and found that there are 

- 10 keymaps which use "stack" layout: default, dvorak, dvorak_nguyenvietyen, farmergreg, heatxsink, jwon, peott-fr, stapelberg, tw1t611, xyverz

- only 3 keymaps which use "split" layout: default_pretty, insertsnideremarks, tuesdayjohn

Other contributors did a lot of work by creating these keymaps and I would like to take advantage of this work, if possible. Also, this puts kintwin controller at a disadvantage because every other controller does define both "layout" and "layout_pretty" layouts.

Reply:
No, `LAYOUT` should be removed, and `LAYOUT_pretty` renamed to just `LAYOUT`.
And yes, that means this PR would be blocked until those keymaps can be updated.

Reply:
so @alvicstep would have to edit all 10 other keymaps created by others (or they fix it on other PR's) before this PR is considered? that seems pretty unreasonable.
how is it that those 10 other keymaps were merged in to begin with. 
if a mistake was merged in, is it then the responsibility of the next PR to address those mistakes?

if a new guideline/rule was instantiated at a later date, isn't it more reasonable to open issues tagging the related parties?

Reply:
Have a look at my PR again.

Reply:
@fauxpark  it's unclear without digging whether your pr makes updates to keymaps that you yourself created.
what is your response to the question above?
by your last comment do you mean to say something like "i made updates to other peoples work, so you can too"?

your PR also only makes changes to existing keymaps (again unclear whether exclusively yours, exclusively not yours, or mix). which seems  more like a code improvement PR /refactor.

the scope of this PR is a new feature addition. 

again the question is: should it be that new PR's for new features have to fix previously introduced errors before they can be merged? that seems both way out of scope for one PR, and a pretty big issue for open PR congestion

Reply:
Yes, it is out of scope. I will sort out the user keymaps.

Reply:
@fauxpark looks like @alvicstep made changes making this conversation obsolete .
can we resolve this conversation and re-review? or are you still in the process of making commits to this branch or separate branch with the above mentioned changes?

Reply:
The keymaps have been updated. #21569

Reply:
@alvicstep this will nullify your last commit and prolly create a few conflicts once you rebase / merge upstream?

Reply:
Based on the changes in [21569](https://github.com/qmk/qmk_firmware/pull/21569) by @fauxpark I have moved kintwin directory back to kinesis directory.

Because 21569 PR was created off of "develop" branch I created new branch from "develop" branch as well and created new PR: [21614](https://github.com/qmk/qmk_firmware/pull/21614). 

@fauxpark , please review

P.S.
I am closing this PR. Hopefully new PR will be more lucky